jbang-catalog.json

```diff
@@ -0,0 +1,12 @@
+{
+  "catalogs": {},
+  "aliases": {
+    "wanaku": {
+      "script-ref": "jbang/src/main/java/ai/wanaku/jbang/WanakuJBang.java",
+      "dependencies": [
+        "ai.wanaku:cli:${wanaku.version:0.0.3}"
```

Comment:
why is it needed? can't you add the dependency here https://github.com/wanaku-ai/wanaku/pull/226/files#diff-74d36cc231abb4699b2017fbb0a2c0b452925d4ddcbb65c9d1b63cd92da876f7R7 ? as in Camel https://github.com/apache/camel/blob/main/dsl/camel-jbang/camel-jbang-main/dist/CamelJBang.java#L23

Reply:
I somehow liked it more this way, so we don't track dependency versions in Java file (i don't like mixing concerns too much). That is the only reason, i could move it to the Java file if you prefer.

Reply:
okok! sounds good, no problem